core/src/test/scala/co.blocke.scalajack/v4/SimpleTests.scala

```diff
@@ -7,10 +7,17 @@ import scala.language.postfixOps
 import scala.util.Try
 import org.joda.time.{ DateTime, DateTimeZone }
 import org.joda.time.format.DateTimeFormat
+import scala.reflect.runtime.universe.{ Type, typeOf }
+import BijectiveFunctions.fullNameToType
 
 class SimpleTestSpec extends FunSpec with GivenWhenThen with BeforeAndAfterAll {
   val sjJS = ScalaJack()
 
+  val hintMod = new HintModifier {
+    def apply(rawHint: String) = fullNameToType.apply("co.blocke.scalajack.test.v4." + rawHint)
+    def unapply(hintFieldType: Type) = fullNameToType.unapply(hintFieldType).split('.').last
```

Comment:
Perhaps...  That was just one case I thunk of.  I really envisioned this being wiiiiide open to whatever logic the user wanted.  I suppose if we think this is still too much for our dear users we could create 2 or 3 canned modifiers/bijectives, including one that is a class name mutator that accepts a String->String function or something.   Not sure.